README.md

```diff
@@ -22,9 +22,9 @@ Generate API docs in Foreman:
 
 Prepare folder for the new version (X.Y)
 
-1. cd to apidocs/foreman directory
+1. cd to foreman directory
 1. `cp -r TEMPLATE X.Y`
-1. `ln -sf X.Y latest`
+1. `unlink latest; ln -s X.Y latest`
```

Comment:
The `-f` option of the `ln` command does not work to forcefully update the symlink. I tried `-nsf` but that did not work too. So thought to use `unlink` instead.

Reply:
My bad, sorry! Good catch.